### ISSUE TITLE
PostCSS website

### ISSUE THREAD

**Issue by ai** [postcss/postcss#391]
Let’s collect ideas about future PostCSS website.

1. Plugins list (even with search like in gulp) is a lame way. We need some wizard (of course list will exists too):
   1. You select basement: preprocessors or/and CSS4.
   2. You select architecture: BEM, CSS Modules, etc.
   3. You select a builder: Gulp, webpack, grunt, etc.
   4. Pick some extra sugar.
   5. We will get you a exactly code to copy-paste to your build process. For example, only `gulp-cssnext` with some sugar plugin additions if you select CSS4, BEM, Gulp and not so many additions.
2. AST explorer for plugin developer. You put a CSS and get PostCSS AST with references.
3. [Migration guide](https://github.com/postcss/postcss/issues/390)
4. Docs with good formation.
5. Playground based on `postcss-use`.

**Comment by ai** [comment]
Also I want a good art work for site. Something like pixelart or/and watercolor. Funny but about black magic.

Can you recommend a good artists to place a order?

**Comment by MadLittleMods** [comment]
@ai [Here is a list of artists](https://gist.github.com/MadLittleMods/e0fa5ed14def5fea1a65) I gathered so far.

**Comment by yisibl** [comment]
Playground +1

**Comment by ai** [comment]
This issue has many good ideas of Plugins section: 333

**Comment by omgovich** [comment]
How about a section where some people will say ideas of plugins?

**Comment by ai** [comment]
@omgovich GitHub issues is a better way to discuss. But we should promote creating plugins issues on website.

**Comment by admdh** [comment]
[THE LOOK] Maybe it would be good idea to introduce PostCSS like something more luxury, more ambitious? Something that make trends and not follows them. For example just look at bootstrap and foundation websites i think they look like 90th now. PostCSS website definitely shouldn't be like that. i think PostCSS website can, or even should be more conceptual.
[Features] i think the most important feature of the website should be well organised documentation, easy to use, easy to find. Maybe search page like gulpjs.com has (as it was mentioned before). I think it would be good if there were rules which plugins will be recommended first (for example those which have good documentation) and which last (with bad documentation). maybe plugins statistics.

**Comment by admdh** [comment]
[LANDING PAGE] prelude. best PostCSS features. landing page should be attractive so it could make developers switch to PostCSS or at list make them think about it.

**Comment by MoOx** [comment]
To follow babel move, I am ready to "offer" the work I did on cssnext website so it become the postcss website. cssnext should be a "simple" preset, and would not require a website.
I will probably deprecate all cssnext-* plugins (grunt/gulp/etc) and same for the website, when postcss will have one in order to provide a good experience for cssnext user.
If you are interest to get cssnext website to start with something, please let me know.

**Comment by himynameisdave** [comment]
Not sure if this is at all helpful but here's an exportable, ever updating list of plugins:

[**postcss-plugins**](https://github.com/himynameisdave/postcss-plugins)

**Comment by simonsmith** [comment]
Could it be an idea to split the production of the site into stages? For example, get a version of the site designed that contained a homepage with an overview, some documentation and perhaps a list of plugins? 

Then iterate on that to add the more advanced features in the first post here. I think if it the site tries to be too awesome from the first launch it may make the task seem too vast.

**Comment by ai** [comment]
@simonsmith Getting Real is always good :)

**Comment by marcustisater** [comment]
Sounds awesome, I would love to help out with this project.

**Comment by ai** [comment]
Guys, who want to be a leader in this project? :)

**Comment by marcustisater** [comment]
I think you should @ai :)

**Comment by ai** [comment]
Right now we have a few tasks, and I think they are more important.

But I can ask some people. Many developers want to help. We need only a leader :D.

**Comment by marcustisater** [comment]
Yeah that would be great, maybe we can have a initial meeting over at gitter next year. I guess not much wont be happening now during christmas holidays...

**Comment by marcustisater** [comment]
I talked to @ai and I can sign up as a "leader/project manager" for this starting after new years. I completely agree with @simonsmith of starting small but then yet again would love to know how many would be offering some time into this so we can get down to planning, maybe we can offer the full package right away for a first release. 

*Who* is interested of putting some time into this and be willing to help out? There are already some great ideas and hungry developers in this issue so a startup meeting or overview planing would be best to start things of.  

Let *me* or *@ai* know by e-mail or right here in this issue if we still have you're attention! :smiley: 

Thanks and happy holidays! :snowman:

**Comment by simonsmith** [comment]
Put my name down for now, I can probably help depending on the tasks :)

**Comment by ai** [comment]
I add a repo for @marcustisater https://github.com/postcss/postcss.com

**Comment by hudochenkov** [comment]
@marcustisater I can help with HTML/CSS and not complicated JS (SPA, React, etc. i can't do). Also I'm not a designer and can't bring design from my head.

**Comment by himynameisdave** [comment]
I would be down to help as well, and it seems like other developers are able to help in the same small capacity that I am (ie: doing some minor HTML/CSS/JS work). 

It would be cool if (once this project gets off the ground) you guys listed concise, singular modules/components that needed to be built as issues in the [`postcss.com`](https://github.com/postcss/postcss.com) repo. That way whoever is available can pick them up and build them out, independent of how the rest of the application is being built.

**Comment by stephenway** [comment]
I can help with design and or frontend work if needed? @ai love your idea of the theme of the artwork.

**Comment by MoOx** [comment]
Don't know what you plan to use for the engine, but I would be really happy to offer some support for my new engine [Statinamic](http://moox.io/statinamic/) (react based static website generator, that also provide client side optimised navigation).

**Comment by marcustisater** [comment]
I would love to talk more about that @MadLittleMods - We need a lead developer too... :wink:

**Comment by jonathantneal** [comment]
@marcustisater, count me available for any frontend work; HTML, schema, complicated vanilla JS, CSS. Feel free to use me as needed. I’m happy to help.

**Comment by mxstbr** [comment]
I've commented in [`postcss/postcss.com`](https://github.com/postcss/postcss.com), but I'll do it here as well:

Available for all frontend work. (Including the wizard, which I imagine similar in functionality to [babels wizard](https://babeljs.io/docs/setup/))
I made [postcss.parts](http://postcss.parts) and can easily transfer that over to the website.

Wouldn't mind taking responsibility either @marcustisater, I've been meaning to contribute to PostCSS in a more meaningful way for quite some time!
